### PR TITLE
Add chrome driver

### PR DESCRIPTION
**Pull request by erickueen**
- Add chromium and chromium-chromedriver to dependencies
- Update elixir to 1.12.2

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -48,6 +48,9 @@ RUN \
       rm -rf /var/cache/apk/* && \
       chmod +x /usr/bin/wkhtmltopdf && \
       apk add g++
+  RUN apk add --no-cache --update \
```
Comment (sebasortiz-dev) [inline Dockerfile:51]: @erickueen please dont add another RUN, that creates another layer in the docker image that we don't need.

Comment (sebasortiz-dev) [reply Dockerfile:51]: Or is there any reason you want this to be another layer?

File: Dockerfile [hunk 2]
```diff
@@ -4,7 +4,7 @@ FROM alpine:3.13
 LABEL maintainer="Jhon Pedroza <jpedroza@cuentamono.com>"
 ENV ERLANG_VERSION=23.2.7
 # elixir 1.12.0
```
Comment (johnf9896) [inline Dockerfile:6]: Update the comment

File: Dockerfile (hunk 1 above)
Comment (johnf9896) [reply Dockerfile:51]: I agree. A previous RUN can be used

**Review by johnf9896 — CHANGES_REQUESTED** [review]

**Review by johnf9896 — APPROVED** [review]

**Pull request merged**